app/models/configuration_profile.rb

```diff
@@ -72,4 +72,8 @@ def tag_hash
   def total_configured_systems
     Rbac.filtered(configured_systems, :match_via_descendants => ConfiguredSystem).count
   end
+
+  def image_name
```

Comment:
Oh, OK then :).. we *may* want to move those to decorators too, but since they're actually being used on undecorated models, probably warrants a separate PR..